core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentStatusManager.java

```diff
@@ -493,6 +493,19 @@ public static void writeLoadDetailsIntoFile(String dataLoadLocation,
             invalidLoadIds.add(loadId);
             return invalidLoadIds;
           }
+          // if the segment status is in progress then no need to delete that.
+          if (SegmentStatus.INSERT_IN_PROGRESS == loadMetadata.getSegmentStatus()) {
+            LOG.error("Cannot delete the Segment which is In Progress. Segment is " + loadId);
```

Comment:
Please rephrase the sentence 
`"Cannot delete the segemnt "+ loadId +" which is load in progress" `